shencha-master/shencha-master/app.py

```
@@ -1,410 +1,410 @@
+                raise e
+            except Exception as e:
+                raise HTTPException(
+                    status_code=400,
+                    detail={
+                        "error": "FILE_PROCESSING_ERROR",
+                        "message": str(e),
+                        "filename": file.filename,
+                        "file_id": file_id,
+                        **({"url": url} if is_url else {})
+                    }
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)